src/api/app/models/workflow_run.rb

```diff
@@ -31,6 +31,9 @@ class WorkflowRun < ApplicationRecord
   has_many :scm_status_reports, class_name: 'SCMStatusReport', dependent: :destroy
   has_many :event_subscriptions, dependent: :destroy
 
+  delegate :workflow_configuration_path, to: :token
+  delegate :workflow_configuration_url, to: :token
```

Comment:
A `WorkflowRun` is happening at a specific time. Tokens are always *now*.

Reply:
True! I have to save the value those fields had when the workflow ran. Changing...